lib/ts/recipe/dashboard/api/userroles/removeUserRole.ts

```diff
@@ -11,11 +12,20 @@ const removeUserRole = async (
 ): Promise<
     | {
           status: "OK";
+          didUserHaveRole: boolean;
       }
     | {
-          status: "UNKNOWN_ROLE_ERROR";
+          status: "UNKNOWN_ROLE_ERROR" | "FEATURE_NOT_ENABLED_ERROR";
```

Comment:
i thought you are not going to do this in any API since it's done in the entry point API?